src/newsblurapi.cpp

```diff
@@ -283,37 +283,37 @@ rsspp::Feed NewsBlurApi::fetch_feed(const std::string& id)
 			json_object* node{};
 
 			if (json_object_object_get_ex(
-					item_obj, "story_title", &node) == TRUE) {
+					item_obj, "story_title", &node) == 1) {
 				item.title = json_object_get_string(node);
 			}
 
 			if (json_object_object_get_ex(
-					item_obj, "story_authors", &node) == TRUE) {
+					item_obj, "story_authors", &node) == 1) {
 				item.author = json_object_get_string(node);
 			}
 
 			if (json_object_object_get_ex(item_obj,
 					"story_permalink",
-					&node) == TRUE) {
+					&node) == 1) {
 				item.link = json_object_get_string(node);
 			}
 
 			if (json_object_object_get_ex(
-					item_obj, "story_content", &node) == TRUE) {
+					item_obj, "story_content", &node) == 1) {
 				item.content_encoded =
 					json_object_get_string(node);
 			}
 
 			const char* article_id{};
 			if (json_object_object_get_ex(item_obj, "id", &node) ==
-				TRUE) {
+				1) {
```

Comment:
I would prefer dropping the ` == 1` altogther, just keeping:
`if (json_object_object_get_ex(item_obj, "id", &node)) {`

Reply:
@dennisschagt I'd agree if this function returned `bool`, but here it returns `int`, so I think it's dangerous to lump all the non-zero values into "true". Let's keep the strict check.